fluentui_controls/src/main/java/com/microsoft/fluentui/tokenized/controls/BasicCard.kt

```diff
@@ -0,0 +1,53 @@
+package com.microsoft.fluentui.tokenized.controls
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.draw.shadow
+import com.microsoft.fluentui.theme.FluentTheme
+import com.microsoft.fluentui.theme.token.ControlTokens
+import com.microsoft.fluentui.theme.token.controlTokens.BasicCardInfo
+import com.microsoft.fluentui.theme.token.controlTokens.BasicCardTokens
+import com.microsoft.fluentui.theme.token.controlTokens.CardType
+
+/**
+ * Cards are flexible containers that group related content and actions together. They reveal more information upon interaction.
+ * A Basic card is a card with an empty container and basic elevation and radius.
+ *
+ * @param modifier Modifier for the card
+ * @param basicCardTokens Optional tokens for customizing the card
+ * @param content Content for the card
+ */
+@Composable
+fun BasicCard(
+    modifier: Modifier = Modifier,
+    basicCardTokens: BasicCardTokens? = null,
+    content: @Composable () -> Unit
+) {
+    val token = basicCardTokens
+        ?: FluentTheme.controlTokens.tokens[ControlTokens.ControlType.BasicCard] as BasicCardTokens
+    val basicCardInfo = BasicCardInfo(CardType.Elevated)
+    val cornerRadius = token.cornerRadius(basicCardInfo = basicCardInfo)
+    val backgroundColor = token.backgroundColor(basicCardInfo = basicCardInfo)
```

Comment:
Lets rename it to background or BackgroundBrush instead of backgroundColor. It will be more clear.

Reply:
Done